packages/curve-ui-kit/src/features/select-token/ui/TokenSelectButton.tsx

```diff
@@ -0,0 +1,95 @@
+import CircularProgress from '@mui/material/CircularProgress'
+import Select from '@mui/material/Select'
+import Stack from '@mui/material/Stack'
+import Typography from '@mui/material/Typography'
+import KeyboardArrowDownIcon from '@mui/icons-material/KeyboardArrowDown'
+import type { SxProps } from '@mui/system'
+
+import { SizesAndSpaces } from '@ui-kit/themes/design/1_sizes_spaces'
+import { TokenIcon } from '@ui-kit/shared/ui/TokenIcon'
+
+const { Spacing, ButtonSize, MinWidth } = SizesAndSpaces
+
+import type { TokenOption } from '../types'
+
+const ButtonContent = ({ token, disabled }: { token: TokenOption; disabled: boolean }) => (
+  <Stack direction="row" gap={Spacing.xxs} alignItems="center">
+    <TokenIcon
+      blockchainId={token.chain}
+      symbol="" // No need for tooltip of symbol as it's displayed right next to it
```

Comment:
```suggestion
      // No need for tooltip of symbol as it's displayed right next to it
```

Reply:
Had to make symbol an optional property that defaults to the empty string for this